src/openms/include/OpenMS/ANALYSIS/OPENSWATH/MRMFeatureFilter.h

```diff
@@ -28,74 +28,71 @@
 // ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 //
 // --------------------------------------------------------------------------
-// $Maintainer: Douglas McCloskey $
-// $Authors: Douglas McCloskey $
+// $Maintainer: Douglas McCloskey, Pasquale Domenico Colaianni $
+// $Authors: Douglas McCloskey, Pasquale Domenico Colaianni $
 // --------------------------------------------------------------------------
 
 #ifndef OPENMS_ANALYSIS_OPENSWATH_MRMFEATUREFILTER_H
 #define OPENMS_ANALYSIS_OPENSWATH_MRMFEATUREFILTER_H
 
+#include <OpenMS/ANALYSIS/MRM/ReactionMonitoringTransition.h>
 #include <OpenMS/ANALYSIS/OPENSWATH/MRMFeatureQC.h>
 #include <OpenMS/ANALYSIS/TARGETED/TargetedExperiment.h>
+#include <OpenMS/CONCEPT/LogStream.h>
+#include <OpenMS/DATASTRUCTURES/DefaultParamHandler.h>
+#include <OpenMS/DATASTRUCTURES/String.h>
+#include <OpenMS/FORMAT/QcMLFile.h>
 #include <OpenMS/FORMAT/TraMLFile.h>
-
-#include <OpenMS/KERNEL/MRMFeature.h>
 #include <OpenMS/KERNEL/Feature.h>
 #include <OpenMS/KERNEL/FeatureMap.h>
-#include <OpenMS/FORMAT/QcMLFile.h>
-
-#include <OpenMS/DATASTRUCTURES/DefaultParamHandler.h>
-#include <OpenMS/DATASTRUCTURES/String.h>
-#include <OpenMS/CONCEPT/LogStream.h>
+#include <OpenMS/KERNEL/MRMFeature.h>
 
 namespace OpenMS
 {
-
   /**
-
-    @brief The MRMFeatureFilter either flags components and/or transitions that do not pass the QC criteria or filters out
-      components and/or transitions that do not pass the QC criteria. 
+    @brief The MRMFeatureFilter either flags or filters out components
+    and/or transitions that do not pass the QC criteria.
 
     @htmlinclude OpenMS_MRMFeatureFilter.parameters
-
   */
   class OPENMS_DLLAPI MRMFeatureFilter :
     public DefaultParamHandler
   {
-
 public:
-
-    //@{
     /// Constructor
     MRMFeatureFilter();
 
     /// Destructor
-    ~MRMFeatureFilter() override;
-    //@}
+    ~MRMFeatureFilter() = default;
 
     /// Synchronize members with param class
     void updateMembers_() override;
 
+    enum C_Metrics : Size {RT = 0, INTENSITY, QUALITY, METAVALUE};
```

Comment:
Not sure what you are trying to accomplish here.

Reply:
This will go away. Please see the comment below where I refer this enum name.